build.gradle

```
@@ -29,6 +29,8 @@ dependencies {
     testImplementation('org.springframework.boot:spring-boot-starter-test') {
         exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
     }
+    compile group: 'org.springframework.boot', name: 'spring-boot-starter-webflux'
```

```suggestion
implementation 'org.springframework.boot:spring-boot-starter-webflux'
```
처럼 사용해서 다른 springframework 들과 통일성 있게 사용하면 좋을 것 같아요 😄